extensions/aria_extension_tosca/simple_v1_0/modeling/__init__.py

```diff
@@ -85,9 +85,14 @@ def create_service_template_model(context): # pylint: disable=too-many-locals,to
     policies = context.presentation.get('service_template', 'topology_template', 'policies')
     if policies:
         for policy in policies.itervalues():
-            if model.policy_types.get_descendant(policy.type).role == 'plugin':
+            role = model.policy_types.get_descendant(policy.type).role
```

Comment:
*All* the policies are added in line 115, below. Well, I think we *have* to add all of them because they are there in the TOSCA service template. The fact that ARIA recognizes some special roles doesn't mean they aren't policies, still.